### ISSUE TITLE
README is wrong about requiring capistrano-middleman

### ISSUE THREAD

**Issue by codener** [fedux-org/capistrano-middleman#5]
First: thank you for this gem! It was the missing link for my Rails workflow ported to Middleman. Deployment is a breeze now.

There always was a small issue: Middleman would build the project twice. Since the deployment worked nevertheless, I ignored it.

When I started to use different environments like `staging` and `production` with Middleman, this turned out to be a greater issue. I added `set :middleman_options, %w[--environment=staging]` to `deploy/staging.rb`. After some debugging I saw that only the second build would receive these opts.

Turns out you **must not require capistrano-middleman manually in the Capfile**. Setting `:scm, :middleman` also loads the gem. Because the archive name is random, this will run two distinct archive tasks.

Bottom line: Please remove the "require it in your Capfile" paragraph from the README. I think this is not worth a PR; however if you need one, I'll make one.

**Comment by maxmeyer** [comment]
Thanks! Removed.
